doc/conf.py

```diff
@@ -70,6 +76,9 @@
 # -- options for HTML output -------------------------------------------------
 html_theme = "furo"
 html_static_path = ["_static"]
+html_css_files = [
+    "css/style.css",
+]
```

Comment:
If you choose to remove the CSS style change, don't forget to remove this 3 lines as well.